content/upgrade-pop.md

```diff
@@ -213,17 +217,17 @@ sudo apt full-upgrade 2>/dev/null | tee ~/upgrade.log
 
 You may want to take a look at the files that end in "list" in "/etc/apt/backup" to see if you want to enable them again by moving them back to the /etc/apt/sources.list.d/ directory.
 
-### 5. After the 20.04 Pop upgrade is complete, **reboot**
+### 5. After the 20.04 Pop upgrade is complete, **Reboot**
 
 ### 6. Run the command to upgrade to the newest release
 
 ```bash
 pop-upgrade release upgrade
 ```
 
-## Upgrading older releases (20.10)
+## Upgrading older releases to 20.04
 
-Like other older releases, Pop!\_OS 20.10 update files are no longer in the Ubuntu mirror network. Getting your system ready for the upgrade does require some work to manually upgrade the system. Use the following steps to upgrade the system:
+On older releases, Pop!\_OS update files are no longer in the Ubuntu mirror network. Getting your system ready for the upgrade does require some work to manually upgrade the system. Use the following steps to upgrade the system:
```

Comment:
Change from `does require` to `requires`. We're not refuting anything to the contrary, so there's no reason to say `does`.

Reply:
Resolved.